src/app/view/perspectives/search/all/search-all.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="mt-2">
-  <ng-container *ngIf="loading()">
+  <ng-container *ngIf="loading()" i18n="@@search.all.loading">
     Loading...
```

Comment:
The same as before. I am afraid that `<ng-container>` is not real HTML element and so it will not be in the DOM. We should not add attributes on it.